webapp/channels/src/components/more_direct_channels/list/index.ts

```diff
@@ -127,6 +131,14 @@ export function makeGetOptions(): (state: GlobalState, users: UserProfile[], val
     );
 }
 
+function mapDispatchToProps(dispatch: Dispatch) {
+    return {
+        actions: bindActionCreators({
```

Comment:
Since the component is a function component, I would prefer to use the hook (`useDispatch`) directly. But 0/5

@hmhealey Thoughts here?